app/controllers/api/service_alerts_controller.rb

```diff
@@ -36,7 +36,7 @@ def create
   end
 
   def update
-    if service_alert.update_attributes service_alert_params
+    if service_alert.update service_alert_params
```

Comment:
[#update_attributes was deprecated in Rails 6](https://blog.saeloun.com/2019/04/15/rails-6-deprecates-update-attributes.html)